### ISSUE TITLE
Launch play.spotify.com / Soundbounce when the extension button is clicked

### ISSUE THREAD

**Issue by andrewthecoder** [pdaddyo/soundbounce#266]
For now it's a functional message which is displayed to tell the user to open play.spotify.com and log in first, but it would be much better if when you clicked the button, it opened spotify for you.

Even better if it could open spotify for you, wait for you to be logged in and everything loaded, then launch Soundbounce!

**Comment by andrewthecoder** [comment]
Extensions window API: https://developer.chrome.com/extensions/windows
However, I reckon it might be nicer to make SB into a Chrome "app" which would give it more control;

**Comment by pdaddyo** [comment]
Yep all good ideas, thanks Andrew. I actually quite SB in a normal tab, so this should be an option for users if we enable launching in a sep window.
